test_util/EclRegressionTest.hpp

```diff
@@ -162,6 +162,8 @@ class ECLRegressionTest: public ECLFilesComparator {
     const std::vector<std::string> keywordsStrictTol = {"COORD", "ZCORN", "PORV", "DEPTH", "DX", "DY", "DZ", "PERMX", "PERMY", "PERMZ", "NTG",
                                                         "TRANX", "TRANY", "TRANZ", "TRANNNC", "SGRP", "SCON", "DOUBHEAD"
                                                        };
+    // keywords that should not be compared
+    const std::vector<std::string> keywordsBlackList = {"TCPU"};
```

Comment:
I would use `std::unordered_set<std::string>` instead of a vector; and then the `.count(keyword)` method to lookup instead of a linear search.